---
layout: default
title: Marine Louarn, PostDoctoral Fellow
---
<div class="blurb">


	<h1>Publications</h1>
	<p></p>
		<h2><i>Regulus</i> infers signed regulatory relations from few samples’ information using discretization and likelihood constraints</h2>   
                <p><strong>Louarn Marine</strong>, Guillaume Collet, Ève Barré, Thierry Fest, Olivier Dameron, Anne Siegel, Fabrice Chatonnet -- <i>PLOS Computal Biololy (2024)</i><br />
		<strong>Abstract</strong>  Gene expression regulation is based on the activity of specialized regulatory proteins called transcription factors (TFs) which can bind DNA at specific sequences. Understanding the regulatory relations between TFs and genes in humans is fundamental in personalized clinical settings, to better decipher the pathological mechanisms and to identify new therapeutic solutions. However, finding the main regulators of such systems is usually difficult, due to the scarcity of available samples and the biological closeness of the studied cell types. To overcome these issues, we introduce a new tool called Regulus. We use information from genes and TFs expression, regulatory regions activity and TF binding sites occurrences to compute TF-gene relations. We then apply a likelihood reasoning step, based on the biological knowledge of transcriptional regulation mechanisms, to select the most probable relations and assign them a function as activation or inhibition. Finally, we reduce the potential TFs list by a specificity / coverage filter and we annotate it according to existing literature. By testing Regulus on large-scale biological datasets, each describing four biological contexts, we show that this tool is able to i) identify both known and undescribed regulators consistent with all the gene expression and region accessibility constraints in each biological context, ii) include low expressed genes in its relations and iii) considerably limit the space of putative TF-gene relations.<br />
		<strong>Index Terms</strong> Bioinformatics, Gene expression, Gene regulation, Synthetic genetic networks, B cell, Cell differentiation, Transcription factors<br />

		<a href="journal.pcbi.1011816.pdf">pdf</a>
                </p>
	
	        <h2>Improving reusability along the data life cycle: a regulatory circuits case study</h2>   
                <p><strong>Louarn Marine</strong>, Chatonnet Fabrice, Garnier Xavier, Fest Thierry, Siegel Anne , Dameron Olivier -- <i>Journal of Biomedical Semantics (2022)</i><br />
		<strong>Abstract</strong>  In life sciences, there has been a long-standing effort of standardization and integration of reference datasets and databases. Despite these efforts, many studies data are provided using specific and non-standard formats. This hampers the capacity to reuse the studies data in other pipelines, the capacity to reuse the pipelines results in other studies, and the capacity to enrich the data with additional information. The Regulatory Circuits project is one of the largest efforts for integrating human cell genomics data to predict tissue-specific transcription factor-genes interaction networks. In spite of its success, it exhibits the usual shortcomings limiting its update, its reuse (as a whole or partially), and its extension with new data samples. To address these limitations, the resource has previously been integrated in an RDF triplestore so that TF-gene interaction networks could be generated with two SPARQL queries. However, this triplestore did not store the computed networks and did not integrate metadata about tissues and samples, therefore limiting the reuse of this dataset. In particular, it does not enable to reuse only a portion of Regulatory Circuits if a study focuses on a subset of the tissues, nor to combine the samples described in the datasets with samples from other studies. Overall, these limitations advocate for the design of a complete, flexible and reusable representation of the Regulatory Circuits dataset based on Semantic Web technologies.<br />
		<strong>Index Terms</strong> Semantic Web, bioinformatics, SPARQL, reproducibility of results<br />

		<a href="s13326-022-00266-4.pdf">pdf</a>
                </p>

                <h2>Increasing life science resources re-usability using semantic web technologies</h2>
                <p><strong>Louarn Marine</strong>, Chatonnet Fabrice, Garnier Xavier, Fest Thierry, Siegel Anne , Dameron Olivier -- <i>eScience (2019)</i><br />
		<strong>Abstract</strong>  In life sciences, current standardization and integration efforts are directed towards reference data and knowledge bases. However, original studies results are generally provided in non standardized and specific formats. In addition, the only formalization of analysis pipelines is often limited to textual descriptions in the method sections. Both factors impair the results reproducibility, their maintenance and their reuse for advancing other studies. Semantic Web technologies have proven their efficiency for facilitating the integration and reuse of reference data and knowledge bases.<br />
		We thus hypothesize that Semantic Web technologies also facilitate reproducibility and reuse of life sciences studies involving pipelines that compute associations between entities according to intermediary relations and dependencies.<br />
		In order to assess this hypothesis, we considered a case-study in systems biology (<a href="http://regulatorycircuits.org">http://regulatorycircuits.org</a>), which provides tissue-specific regulatory interaction networks to elucidate perturbations across complex diseases. Our approach consisted in surveying the complete set of provided supplementary files to reveal the underlying structure between the biological entities described in the data.We relied on this structure and used Semantic Web technologies (i) to integrate the Regulatory Circuits data, and (ii) to formalize the analysis pipeline as SPARQL queries.<br />
		Our result was a 335,429,988 triples dataset on which two SPARQL queries were sufficient to extract each single tissues pecific regulatory network.<br />
		<strong>Index Terms</strong> Semantic Web, bioinformatics, SPARQL, reproducibility of results<br />

		<a href="PID6102547.pdf">pdf</a>
                </p>

                <h2>Constraint-based verification of a modile app-game designed for nudging people to attend cancer screening</h2>
                <p>Gotlieb Arnaud, <strong>Louarn Marine</strong>, Nygard Mari, Ruiz-López Tomas and Sen Sagar -- <i>AAAI, deployed application in Artificial Intelligence (2016)</i><br />
		<strong>Abstract</strong> In Norway, cervical cancer prevention involves the participation of as many eligible women aged 25-69 years as possible. However, reaching and inviting every eligible women to attend cervical cancer screening and HPV vaccination is difficult. Using social nudging and gamification in modern means of communication can encourage the participation of unscreened people. Simula Research Laboratory together with the Cancer Registry of Norway have developed FightHPV, a mobile app game intended to inform adolescent and eligible women about cervical cancer screening and HPV vaccination while they play and, to facilitate their further participation to prevention campaigns. However, game design and health information transfer can be hard to reconcile, as the design of each game episode is more guided by the release of information than gameplay and playing difficulty. In this paper, we propose a constraint-based model of FightHPV to evaluate the difficulty of each episode and to help the game designer in improving the player experience. This approach is relevant to facilitate social nudging of eligible women to participate to cervical cancer screening and HPV vaccination, as shown by the initial deployment of FightHPV and tests performed in focus groups. The design of this mobile app can thus be regarded as a new application case of Artificial Intelligence techniques such as gamification and constraint programming.<br />
		<a href="14432-66198-1-PB.pdf">pdf</a>
		</p>


	<h1>Posters</h1>
	<p></p>
		<h2>Regulus infers signed, context-dependent and process-based regulatory circuits between few cell types</h2>
                <p><strong>Louarn Marine</strong>, Guillaume Collet, Ève Barré, Thierry Fest, Olivier Dameron, Anne Siegel, Fabrice Chatonnet -- <i> Presented in Las Vegas during RSGDREAM (2022)</i><br />
		<a href="Poster_dream_size.pdf">pdf</a>
		<h2>Barriers to detecting intracellular transcriptional regulatory signals in co-expression</h2>
                <p>Eric Ching-Pan Chu, Alexander Morin, <strong>Marine Louarn</strong>, Paul Pavlidis -- <i> Presented in Las Vegas during RSGDREAM (2022)</i><br />
		<a href="iscb-prod-mp4sposter_ericchu_1666851482467.pdf">pdf</a>
		<h2>Increasing life science resources re-usability using semantic web technologies</h2>
                <p><strong>Louarn Marine</strong>, Chatonnet Fabrice, Garnier Xavier, Fest Thierry, Siegel Anne , Dameron Olivier -- <i> Presented in Marseille during the Spring School: Networks and Molecular Biology (2020)</i><br />
		<a href="LifeScience.pdf">pdf</a>
                <h2>DICOMisation matérielle d'un signal analogique: CANDOS</h2>
                <p><strong>Louarn Marine</strong>, Cheron Lucile, Jezequel Yannick -- <i> Presented in Rennes (2016)</i><br />
		<a href="Poster_CANDOS.pdf">pdf</a>
</div><!-- /.blurb -->
