---
layout: default
title: Marine Louarn, PhD student
---
<div class="blurb">
	<h1>bla</h1>
	<p>bla <a href="/about">Read more about my life...</a>
	<a href="http://regulatorycircuits.org/">Regulatory Circuits</a> is a resource of 394 tissue-specific gene regulatory networks for human. We propose here a RDF version of this resource, input data, pipeline and results, queryable through the virtuoso endpoint. We give example of queries in the <a href="https://github.com/mlouarn/RCsparql">github</a> associated to the project. The turtles files are also available.<br />
In a preliminary work, we integrated the samples data into a RDF triplestore and showed that two SPARQL queries could generate the corresponding networks :<br />
 Louarn, M., Chatonnet, F., Garnier, X., Fest, T., Siegel, A., Dameron, O.: <b>Increas-ing life science resources re-usability using semantic web technologies.</b> In: Proceed-ings of the 15th IEEE International eScience conference, San Diego (2019)<br />

The RDF version of the dataset is under the <a href="https://creativecommons.org/licenses/by/4.0/"> Attribution 4.0 International (CC BY 4.0)</a> licence.<br />


List of publication reusing Regulatory Circuits results (with more than 15 citations themselves), that motivated this RDFisation:
<ol>
<li>Gao, L., Uzun, Y., Gao, P., He, B., Ma, X., Wang, J., Han, S., Tan, K.: <b> Identifying noncoding  risk variants  using  disease-relevant  gene  regulatory  networks.</b>   Naturecommunications 9(1), 1–12 (2018)</li>
<li>Horn, H., Lawrence, M.S., Chouinard, C.R., Shrestha, Y., Hu, J.X., Worstell, E.,Shea, E., Ilic, N., Kim, E., Kamburov, A., et al.: <b> Netsig: network-based discovery from cancer genomes</b> . Nature methods 15(1),  61 (2018)</li>
<li>Jia, P., Han, G., Zhao, J., Lu, P., Zhao, Z.: <b> Szgr 2.0: a one-stop shop of schizophrenia candidate genes.</b>  Nucleic acids research45(D1), D915–D924 (2017)</li>
<li>Lomberk, G., Blum, Y., Nicolle, R., Nair, A., Gaonkar, K.S., Marisa, L., Mathison,A., Sun, Z., Yan, H., Elarouci, N., et al.: <b> Distinct epigenetic landscapes underlie the pathobiology of pancreatic cancer subtypes.</b>  Nature communications 9(1), 1–10 (2018)</li>
<li> Lòpez, C., Kleinheinz, K., Aukema, S.M., Rohde, M., Bernhart, S.H., Hübschmann,D., Wagener,  R.,  Toprak,  U.H.,  Raimondi,  F.,  Kreuz,  M.,  et  al.: <b> Genomic  and transcriptomic changes  complement  each  other  in  the  pathogenesis  of sporadic burkitt lymphoma.</b> Nature communications 10(1), 1–19 (2019)</li>
<li>Noh,  H.,  Shoemaker, J.E.,  Gunawan,  R.:  <b> Network  perturbation  analysis  of  gene transcriptional profiles reveals protein targets and mechanism of action of drugs and influenza a viral infection.</b>  Nucleic acids research46(6), e34–e34 (2018)</li>
<li>Paul, D.S., Teschendorff, A.E., Dang, M.A., Lowe, R., Hawa, M.I., Ecker, S., Beyan,H., Cunningham, S., Fouts, A.R., Ramelius, A., et al.: <b> Increased dna methylation variability in type 1 diabetes across three immune effector cell types.</b>  Nature communications 7(1), 1–11 (2016)</li>
<li>Saelens, W., Cannoodt, R., Saeys, Y.: <b> A comprehensive evaluation of module detection methods for gene expression data.</b>  Nature communications 9(1), 1–12 (2018)</li>
<li>Saelens,  W.,  Cannoodt,  R.,  Todorov,  H.,  Saeys,  Y.:  <b> A  comparison  of  single-cell trajectory inference methods.</b> Nature biotechnology37(5), 547–554 (2019)</li>
<li>Vosa,  U.,  Claringbould,  A.,  Westra,  H.J.,  Bonder,  M.J.,  Deelen,  P.,  Zeng,  B.,Kirsten, H., Saha, A., Kreuzhuber, R., Kasela, S., et al.: <b> Unraveling the polygenic architecture of complex traits using blood eqtl meta-analysis.</b>  BioRxiv p447367 (2018)</li>
</ol> </p>
</div><!-- /.blurb -->
